### PR TITLE
Update 2019-04-23-bilibili.md

### PR DESCRIPTION
**Pull request by nosoyyo**
Fix typos and some of the misuses of punctuations.

### CODE REVIEW

**Comment by purerosefallen** [comment]
How broken is Bilibili's English skills! I doubt if they had performed any worldwide actions...

**Comment by binarywang** [comment]
感觉光改些错别字没什么改善，还有些语法错误之类的，整体需要重写。

**Comment by expyhist** [comment]
这英语水平……后端变量名不知道会不会写错🤔

**Comment by awm1599** [comment]
哈哈哈，I think so……

File: 2019/04/2019-04-23-bilibili.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
-Hello Dear Github:
```
Comment (fatur0211) [inline 2019/04/2019-04-23-bilibili.md:1]: Butuh bntuan

**Comment by jessephus** [comment]
Ouch. 

I assume this is just a joke. But as [noted above](https://github.com/github/dmca/pull/5146#issuecomment-486895416), we just re-post the text we receive in DMCA notices here. Because of that, we [do not accept contributions](https://github.com/github/dmca/blob/master/CONTRIBUTING.md) to change notices.

**Pull request closed without merging**